images/k8s-infra/Dockerfile

```diff
@@ -18,23 +18,23 @@
 # infrastructure managed by wg-k8s-infra.
 
 # base image
-FROM debian:buster
+FROM debian:bullseye
 
 # build args
 
 # unless otherwise stated, versions were arbitrarily chosen to whatever was
 # latest at the time, and package-based installs from debian-maintained repos
 # are not pinned to a specific version
 
-ARG CONFTEST_VERSION=0.25.0
-ARG GCLOUD_VERSION=347.0.0
-ARG GH_VERSION=1.12.1
-ARG GO_VERSION=1.16.6
+ARG CONFTEST_VERSION=0.27.0
+ARG GCLOUD_VERSION=355.0.0
+ARG GH_VERSION=2.0.0
+ARG GO_VERSION=1.17
```

Comment:
I'm not sure the group reconciler has been proven to work with 1.17 yet, I'll have a PR incoming shortly

Reply:
Opened https://github.com/kubernetes/k8s.io/pull/2687